README.md

```diff
@@ -5,13 +5,44 @@ Run an open-source project? Submit a Pull Request to add yourself to the list!
 
 Visit the website: [up-for-grabs.net](http://up-for-grabs.net/)
 
+## Add your project
+
+Each of the projects is a file in the [projects](https://github.com/dahlbyk/up-for-grabs.net/blob/gh-pages/_data/projects/) folder - to add yours, create a new file named after your project. Ensure all spaces and special characters are replaced with `-`, to make everyone's life easier.
+
+The contents of the file are just some details about the project:
+
+```yaml
+name: *your project here*
+desc: *some details about the project*
+site: *home page or repository URL*
+tags: [ *tags*, *to*, *search*, *on* ]
+upforgrabs:
+  name: *the label associated with your tasks*
+  link: *link to the related tags*
```

Comment:
should be "link to the related up-for-grabs label" to avoid confusion with the `tags` key above.

Maybe also link to one of the existing files as an example?